projects/client/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -39,19 +43,14 @@
     <DefineConstants>$(DefineConstants);CORECLR</DefineConstants>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
-    <OutputType>Library</OutputType>
     <Optimize>true</Optimize>
-    <AssemblyOriginatorKeyFile>../../../rabbit.snk</AssemblyOriginatorKeyFile>
-    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' != 'Release' ">
+    <Optimize>false</Optimize>
```

Comment:
This isn't needed, so you an remove this entire PropertyGroup and the `Optimize` setting.